extensions/amp-accordion/1.0/amp-accordion.css

```diff
@@ -29,3 +29,20 @@
   padding-right: 20px;
   border: solid 1px #dfdfdf;
 }
+
+/* TODO: This CSS is duplicated from ./accordion.jss.js for purposes of display
+ * locking.  Look into de-duping once experiment is launched.
+ */
+amp-accordion.i-amphtml-display-locking > section:not([expanded]) > :last-child,
+amp-accordion.i-amphtml-display-locking
+  > section:not([expanded])
+  > :last-child
+  * {
+  content-visibility: hidden-matchable !important;
```

Comment:
Does this not kill the animation?

Reply:
Confirmed. Animations are broken:

- In Preact collapsing animation is broken
- In AMP expanding animation is broken

Reply:
Fixed animations in the latest revision: https://github.com/ampproject/amphtml/pull/31227/commits/aa5a335db4af265be3c1011c10909307d4789818

The strategy is to add a class `i-amphtml-animating` to the content element while the animations are occuring.  CSS for hidden content will ignore this class `(:not(.i-amphtml-animating))`.